Bottom: 938b7b3c8fbeb5be67b89fe70f57bd9531bf1282
Top:    8964793ffc031d1754f0da46fef51db4d3443455
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-26 23:38:01 +0100

Add support for a two soldermask layers, defining regions of removed mask

File-format wise, these work similarly to how silk layers are supported now:

Layer (1 to n "[NAME]")     # "ordinary" design layers
Layer (n+1 "silk")          # top side
Layer (n+2 "silk")          # bottom side
Layer (n+3 "soldermask")    # top side
Layer (n+4 "soldermask")    # bottom side


---

diff --git a/src/const.h b/src/const.h
index 42a04db..193ebc8 100644
--- a/src/const.h
+++ b/src/const.h
@@ -48,6 +48,8 @@
  */
 #define BOTTOM_SILK_LAYER       0
 #define TOP_SILK_LAYER          1
+#define BOTTOM_SOLDERMASK_LAYER 2
+#define TOP_SOLDERMASK_LAYER    3
 
 /* ---------------------------------------------------------------------------
  * misc constants
